Dockerfile

```diff
@@ -84,6 +88,7 @@ RUN git config --global --add safe.directory /fides
 # Enable detection of running within Docker
 ENV RUNNING_IN_DOCKER=true
 
+USER fidesuser
```

Comment:
yes - now doing something similar in latest commit!

Reply:
oh dear this breaks everything

Reply:
actually it seemed to be the user id change!

Reply:
OK added change ownership for fides directory. I'm not sure if this matters since I think the important thing is that we are running the application as fidesuser, it is fine if root installs everything generally, but added for consistency like you said.